docs/03-dev-howtos/18-apply-the-special-report-tone.md

```diff
@@ -0,0 +1,24 @@
+# Apply the special report tone
+
+Sometimes CP or someone from Editorial will ask us to apply the special report treatment to a tag.
+The following doc explains how to make that happen.
+
+1. Find out the id of the tag where we need the special report tone. For example "news/series/panama-papers".
+2. Replace <NEW-SPECIAL-TAG> with the string of the special tag and run this code on a python REPL:
+```
```

Comment:
It wasn't immediately obvious to me that "the id of the tag" and "the string of the special tag" meant the same thing. Perhaps use the same wording in both cases?